orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -411,12 +417,15 @@ public void changePrimaryGateway(GatewayConfig formerGateway, GatewayConfig newP
 
             // salt '*' state.highstate
             runNewService(sc, new HighStateRunner(server, allNodes), exitCriteriaModel);
-        } catch (Exception e) {
+        } catch (ExecutionException e) {
```

Comment:
this change is unnecessary because the log msg is the same, the throw is the same

Reply:
One less `e instanceof` is the reason of this change